stdlib/builtins.pyi

```diff
@@ -804,7 +804,7 @@ class bytearray(MutableSequence[int], ByteString):
     def __alloc__(self) -> int: ...
 
 @final
-class memoryview(Sequence[int]):
+class memoryview(Sized, Sequence[int]):
```

Comment:
Is this change necessary? I would prefer removing explicit protocol base classes if possible, since memoryview doesn't inherit from Sized at runtime.

Reply:
I'm not sure, I was following would I thought was the consensus after I saw https://github.com/python/typeshed/issues/9296#issuecomment-1332377034 (which suggested including this reversion), and then 10 days having gone by without a differing suggestion.

Can you tell me how if you want me to try undoing this and verifying that the regression is still fixed locally before I push again? I tried adding some garbage to the regression test here locally and then re-running `python tests/pyright_test.py` but it still passed, so I'm not sure how to check this.

Reply:
Let's just undo this change and see if CI succeeds.

```suggestion
class memoryview(Sequence[int]):
```

Reply:
Thanks, @JelleZijlstra. Btw, looks like the message of the commit that you merged to main still includes `Revert "typing: remove metaclass from Sized (#9058). This reverts commit a3ce51209544d64d03018b3927051375f1bebe82.`

More importantly, I would still like to know how to check if the regression test I wrote is actually doing its job: Did the tests that passed here in CI actually exercise the code path where this bug was observed? I'm concerned that when I changed "MyKeysView.register" to "MyKeysView.garbage" and tried to have pyright test that code, I didn't see any failures.